helm/cilium/templates/giantswarm/allow-all-egress-np.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumClusterwideNetworkPolicy
+metadata:
+  name: allow-all-egress
```

Comment:
This should have standard app labels IMO.

Reply:
Maybe. The rest doesn't because it comes from the upstream chart. I'm not really sure what to do with all of that. It would be really nice to use kustomize for that. It's designed for such things.